webpack/ForemanInventoryUpload/Components/PageHeader/components/CloudConnectorButton/CloudConnectorButton.js

```diff
@@ -10,26 +10,26 @@ export const CloudConnectorButton = ({ status, onClick, jobLink }) => {
       <Popover
         bodyContent={
           <div>
-            {__('Cloud connector job is still running, you can view it here:')}
+            {__('Cloud connector setup is in progress now: ')}
```

Comment:
The popover is triggered only by a click,
though I think from a user perspective when they see it's in progress,
they will not click on it - therefore they will not see the job link.

I would suggest:
- disabling the button while it's pending 
- showing the Popover by hovering the button